Autopilot/AttitudeManager/Inc/ultrasonic.hpp

```diff
@@ -0,0 +1,9 @@
+#IFNDEF ULTRASONIC_HPP
```

Comment:
idk why the ultrasonic file has been added, but I don't want to remove it and re-push since our pushes with Travis are limted, I'll remove it from the PR with the other requested changes the very end if that's ok

Reply:
nah feel free to remove it if it will make reviewing easier for us

Reply:
Also, please don't capslock preprocessors 

`#ifndef` and `#define`